platform/ios/Info.plist Keys.md

```diff
@@ -0,0 +1,15 @@
+# Info.plist Keys
+
+The SDK supports several custom `Info.plist` keys in your application in order to configure various settings. 
+
+## MGLMapboxAccessToken
+
+Use this key to set your Mapbox API [access token](https://www.mapbox.com/help/define-access-token/). This key is required for all map use. 
```

Comment:
Also link to `+[MGLAccountManager setAccessToken:]` as an alternative for developers who prefer to set their access token in code, as well as [this guide](https://www.mapbox.com/help/ios-private-access-token/) for those who want to keep their access token super private.